### DISCUSSION TITLE
Navigation in landscape orientation

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#242] [Q&A] [answered]
We can discuss here ideas for a new (probably extra / optional) navigation interface for landscape orientation.

**Comment by devemux86** [comment]
Since this also applies to my case, I'm intrigued. Lately I've been noticing in many navigators a lack of symmetry in placing your horizontal location on the screen. Are you also planning something similar? I admit that it is difficult for me to convince myself of such a screen layout (although I can understand the rationale).

_Originally posted by @Ciutnik in https://github.com/devemux86/cruiser/discussions/9#discussioncomment-6455151._

**Reply by devemux86** [reply]
They put all the panels on one side and the user location on the other side.
This could also be an idea.

There is not anything final. We could discuss any useful UI we can think.

**Comment by airald** [comment]
I also use Cruiser in landscape mode. Actual layout is so good in my oppinion. Info panels on the right (my screen is full since I'm using average speed). On the left there're some free space. Maybe could be used to place info for close POIs (I like a lot the way to do that of TomTom). Location is in the middle, that's more or less in the middle of the available screen.

**Reply by devemux86** [reply]
> On the left there're some free space. Maybe could be used to place info for close POIs (I like a lot the way to do that of TomTom).

There will be settings where to place the nearby POI and favorites (right or left):

<img src="https://github.com/devemux86/cruiser/assets/3484020/579dc8a4-b849-4d59-b293-4fb828ba0f09" width="50%"/>

**Reply by DolomitesTom** [reply]
what happens if there is not enough space, especially in landscape mode?

**Reply by devemux86** [reply]
@DolomitesTom currently nothing special.

There have been some ideas, for example see here: https://github.com/devemux86/cruiser/discussions/9#discussioncomment-6272446.

**Comment by devemux86** [comment]
Will the long list be scrollable in landscape mode?

_Originally posted by @DolomitesTom in https://github.com/devemux86/cruiser/discussions/492#discussioncomment-9102053._

**Reply by devemux86** [reply]
Not at the moment, we can think of alternatives.

For example, Google Maps and some apps in landscape split the screen in half ,
the info panels on the left half screen and the navigation on the right half screen.

Not sure if this is something we want, instead of navigation in the center of the screen,
but it is possible to provide it in the future.

**Reply by airald** [reply, marked as answer]
The result with Cruiser is more or less the same, panels on both sides. About 1/3 of the screen is full of info panels in my setup. I preffer to have panels on the both sides, it's easier to read than all panels on one side, and route are centered on the motorcycle.
